packages/make-styles/tsconfig.json

```
@@ -1,21 +1,17 @@
 {
+  "extends": "../../tsconfig.base.json",
+  "include": ["src"],
   "compilerOptions": {
+    "target": "ES5",
+    "module": "CommonJS",
+    "lib": ["es5", "dom", "ES2015.Core", "ES2015.Proxy", "ES2015.Symbol", "ES2015.Symbol.WellKnown"],
```

I'd narrow this just to `ES2015`, it doesn't make sense to have just those particular feature sets as if one of those is supported in particular browser whole es2015 is supported

```suggestion
    "lib": ["ES2015", "dom"],
```

I'd probably consider also directly setting target to `ES2015`, with this setup we are shipping stuff that might not work in es5 without polyfills

---

Makes sense to me, applied it 👍